Library/Homebrew/cmd/audit.rb

```diff
@@ -182,6 +186,10 @@ def audit_file
       problem "'__END__' was found, but 'DATA' is not used"
     end
 
+    if text =~ /inreplace .* do/ && text.has_one_sub?
```

Comment:
I also noticed now that we should only catch `gsub!` instead of `gsub` for the similar reason.